app/controllers/topological_inventory/ingress_api/v0/inventory_controller.rb

```diff
@@ -5,25 +5,18 @@
 module TopologicalInventory
   module IngressApi
     module V0
-      class InventoryController < ApplicationController
-        skip_before_action :verify_authenticity_token
+      class InventoryController < ::ApplicationController
```

Comment:
That was actually the funny part. This controller was inherited from TopologicalInventory::ApplicationController from core :)

Reply:
Oh wow nice catch!